src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <>
+      <div className="CommentInfo">
+        <div className="CommentInfo__title">
+          <strong className="CommentInfo__name">{name}</strong>
+
+          {' by '}
```

Comment:
it's better to use `span` tag and set some margin to avoid using like this `{' by '}`